spec/system/consumer/shopping/products_spec.rb

```diff
@@ -83,8 +83,14 @@
 
         visit shop_path
         expect(page).to have_content product.name
+
+        # It truncates a long product description.
+        within_product_description(product) do
+          expect(html).to include product.description
```

Comment:
I brought this line back but it's actually wrong because we want to see only part of the description with `...` at the end. It'll be fixed in a later commit.